crs_application/src/task_managers/motion_planning_manager.cpp

```diff
@@ -438,6 +438,11 @@ common::ActionResult MotionPlanningManager::planMediaChanges()
     const trajectory_msgs::msg::JointTrajectory& next_traj = result_.process_plans[i + 1].process_motions.front();
     if (!next_traj.points.empty())
     {
+      if (!result_.process_plans[i + 1].start.points.empty())
```

Comment:
Should the logic not be reversed here?

Reply:
I'm doing 2 checks, first to make sure there is a next trajectory, which there always should be unless the media change is triggered after the last waypoint, and then second making sure there is a start trajectory for said process, which again there always should be because that's how I have the motion planner setup. I feel like most critical there is a process path there even if it doesn't have a start, so I check that first, and then if there is a start I am going to replace it with the return from media change motion, so I then remove the start trajectory.

Does this make sense?